### ISSUE TITLE
Small window pops up before main window

### ISSUE THREAD

**Issue by ExodusHour** [TomSchimansky/CustomTkinter#960]
A small window pops up just before the main window is shown. I don't have this problem with regular Tkinter, so not sure what's causing it. I've tried with the example files and it happens with those as well. I'm on the most recent version, have tried reinstalling, tried on a different PC, and tried in a venv and have had no luck. Found this Stack Overflow answer but it doesn't seem to apply here as all I have is a small base app. https://stackoverflow.com/questions/55890931/python-tkinter-small-window-pops-up-momentarily-before-main-window. Any help would be great, thanks

Edit: Seems to be a problem with 5.0.3, I installed 5.0.2 and I don't have this issue anymore.

[ctk.webm](https://user-images.githubusercontent.com/54953078/209761620-4fff1b29-814b-4bc1-8af3-bfbfb6c983d6.webm)

**Comment by harshnagar** [comment]
Actually the Default Icon of the CustomTkinter is causing this issue for now we can solve this by:
Editing the __init__.py file in tkinter folder of Python, just add # in these Lines:

before:
<img width="593" alt="prev" src="https://user-images.githubusercontent.com/35253819/210134029-6531b504-e22d-469d-bb0d-457acf2f1418.png">

after:
<img width="632" alt="new" src="https://user-images.githubusercontent.com/35253819/210134037-9298a22e-491d-41ca-921d-c460b3fd5ed8.png">

Hope this work for you as it is working for me I know this is not good method but if you can wait for Customtkinter new update to resolve this issue you can use this till then!

**Comment by thomasrice** [comment]
Thanks, this was a good workaround!

**Comment by lachesis17** [comment]
@ExodusHour I've also been encountering this in the latest version 5.0.3.

I'm not sure exactly where the problem is but I know its when creating a customtkinter.CTk() window, which is inside the ctk_tk.py script of the the customtkinter package.

I found some of the issues you described related to wm_iconbitmap() in other questions but couldn't apply this to customtkinter windows.

https://stackoverflow.com/questions/55890931/python-tkinter-small-window-pops-up-momentarily-before-main-window
https://stackoverflow.com/questions/34156037/python-tkinter-small-window-pops-up-momentarily-before-main-window

A workaround is to create your main window as a regular tkinter.Tk() window instead of a customtkinter window, and then create a customtkinter.CTkFrame() inside the main window so you can still use all the customtkinter methods and dependencies.

Example:

```
import tkinter as tk
import customtkinter as ct

class AppFrame(ct.CTkFrame):

    def __init__(self):
        super(AppFrame, self).__init__(main_window)

        main_window.geometry('250x100+600+200')
        main_window.title("Main Window")
        
        self.button = ct.CTkButton(self, text="Button!", command=self.button_command, fg_color="#000000", corner_radius=5, hover_color="#27af42", text_color="#FFFFFF", font=("Tahoma",11), width=15)
        self.button.pack(pady=8, padx=8)

        self.pack(pady=25)

    def button_command(self):
        print('hello world')

if __name__ == '__main__':
    main_window = tk.Tk()
    app_frame = AppFrame()
    main_window.mainloop()
```

**Comment by TomSchimansky** [comment]
Fixed now with version 5.0.4.
